enterprise/server/test/integration/remote_execution/remote_execution_test.go

```diff
@@ -584,6 +585,27 @@ func TestManySimpleCommandsWithMultipleExecutors(t *testing.T) {
 	}
 }
 
+func TestRedisAvailabilityMonitoring(t *testing.T) {
+	flags.Set(t, "remote_execution.enable_redis_availability_monitoring", "true")
```

Comment:
I would like to add a test that tests this behavior by invoking Bazel, but it turned out to be a little bit more involved than expected. I will follow up with a separate PR so that I can let this change soak in dev.